app/assets/javascripts/lajoya.js

```diff
@@ -81,6 +85,48 @@
     });
   }
 
+  function sendWaveToPost(blob) {
+    var data = new FormData();
+    data.append("audio", blob, (new Date()).getTime() + ".mp3");
+
+    var oReq = new XMLHttpRequest();
+    oReq.addEventListener("progress", updateProgress, false);
+    oReq.addEventListener("load", transferComplete, false);
+    oReq.open("POST", "/uploads");
+    oReq.setRequestHeader('X-CSRF-Token',
+      "xobbGEAxrKTnjkzCcIBB69Df391+sMX6pcqp500R9jmutFCeAv69tCVbngovFHvHaiizJRXDP8R1ugfNxdC61Q");
+    oReq.send(data);
+    var startTime = (new Date()).getTime()
+    console.log("Upload started")
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>'console' is not defined.